src/systemverilog/systemverilog.ts

```diff
@@ -411,8 +411,8 @@ export const language = <ILanguage>{
 		],
 
 		include: [
-			[/(\s*)(<)([^<>]*)(>)/, ['', 'keyword.directive.include.begin', 'string.include.identifier', { token: 'keyword.directive.include.end', next: '@pop'}]],
```

Comment:
we just talked about this but i think this can all be grouped into one regex

Reply:
yep, got it!